### PR TITLE
Seedpostgres

### PR DESCRIPTION
**Pull request by hannahsohkim**
Using csv-write-stream to write 10M records to a csv file. 

No questions for now but please let me know if you see anything that could be better optimized or written. Thanks for reviewing!

### CODE REVIEW

**Review by codysmith287 — COMMENTED** [review]
Overall everything looks good to me. I would say if you're generating all 10 million in 25-30 seconds you have the generation pretty well optimized. It's the fastest I've heard so far. The createTags function  is a little hard to understand what is going on at first but the logic is sound. You might consider renaming some of the variables to make it easier to understand or adding a one or two line comment explaining the function. Not a huge deal though. Great work!

File: generateData.js [hunk 1]
```diff
@@ -26,15 +25,6 @@ const createAlbum = () => ({
   description: faker.lorem.sentences(2),
 });
```
Comment (codysmith287) [inline generateData.js]: This has an implicit return of the object, right? (lines 20-26)

File: generateData.js [hunk 2]
```diff
@@ -2,7 +2,6 @@ const fs = require('fs');
 const csvWriter = require('csv-write-stream');
 let writer = csvWriter();
 const faker = require('faker');
-// let counter = 0;
 
 const createTags = () => {
```
Comment (codysmith287) [inline generateData.js]: Took a minute to understand what was going on with this function but I like your logic here in the createTags function.

Comment (hannahsohkim) [reply generateData.js]: @codysmith287 thanks for letting me know! I went back and added a comment to explain the function and also renamed some of the variables.

File: db/generateData.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  name: faker.random.word() + faker.lorem.word(),
+  artist: faker.name.firstName(),
+  image: faker.image.abstract(),
+  tags: createTags(),
+  description: faker.lorem.sentences(2),
+});
+
+const createAllAlbums = () => {
+  writer.pipe(fs.createWriteStream('db/data.csv'));
+  for (let i = 0; i <= 1000000; i++) {
```
Comment (Druthi) [inline db/generateData.js:32]: Hey! Isn't this supposed to be 10M ?

File: db/generateData.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    const album = createAlbum();
+
+    writer.write({
```
Comment (Druthi) [inline db/generateData.js:35]: This is interesting. Is there a reason you did not write this as, writer.write(album); ?

File: package.json [hunk 5]
```diff
@@ -9,9 +9,10 @@
     "node": ">=6.13.0"
   },
   "scripts": {
-    "start": "nodemon",
+    "start": "nodemon --max-old-space-size=8192 db/generateData.js",
```
Comment (Druthi) [inline package.json:12]: Hey! So are you generating the csv everytime you run the server?

File: schema.sql [hunk 6]
```diff
@@ -0,0 +1,17 @@
+drop database if exists bandland;
```
Comment (Druthi) [inline schema.sql:1]: I was looking to do this approach initially! Will definitely try this out. This is like how I made my mysql database with a schema.sql file.

**Pull request merged**